File: 2356-largest-combination-with-bitwise-and-greater-than-zero/2356-largest-combination-with-bitwise-and-greater-than-zero.cpp
class Solution {
public:
    int largestCombination(vector<int>& candidates) {
        int maxi = 0;
        for(int i = 0; i< 24; i++)
        {
            int cnt = 0; 
            for(int j =0; j< candidates.size(); j++)
            {
                if(((candidates[j])&(1<<i)) !=0)
                    cnt++;
            }
            maxi = max(cnt, maxi);
        }return maxi;



    }
};

File: 2356-largest-combination-with-bitwise-and-greater-than-zero/2356-largest-combination-with-bitwise-and-greater-than-zero.cs
public class Solution {
    public int LargestCombination(int[] candidates) {

        int maxi = 0;
        for(int i = 1; i<= 8388608; i*=2)
        {
            int cnt = 0; 
            for(int j =0; j< candidates.Length; j++)
            {
                if(((candidates[j])&(i)) >0)
                    cnt++;
            }
            maxi = int.Max(cnt, maxi);
        }return maxi;

        
    }
}